openedx/core/djangoapps/oauth_dispatch/tests/test_dot_adapter.py

```diff
@@ -38,6 +43,16 @@ def setUp(self):
             redirect_uri=DUMMY_REDIRECT_URL,
             client_id='confidential-client-id',
         )
+        self.restricted_client = self.adapter.create_confidential_client(
+            name='restricted app',
+            user=self.user,
+            redirect_uri=DUMMY_REDIRECT_URL2,
+            client_id='restricted-client-id',
+        )
+        restricted_app = RestrictedApplication.objects.create(application=self.restricted_client)
+        self.assertEqual(unicode(restricted_app), u"<RestrictedApplication '{name}'>".format(
```

Comment:
Typically we test assertions within a test method outside of the `setUp` method.  Can you move this to a (tiny) test method?